docs/reference/remap2/expressions/arithmetic.cue

```diff
@@ -0,0 +1,161 @@
+package metadata
+
+remap2: expressions: arithmetic: {
+	title: "Assignment"
+	description: """
+		An "arithmetic" expression performs an operation on two expressions as defined by the operator.
+
+		Although arithmetic is commonly performed with numbers, it can apply to other types as well, such as strings.
```

Comment:
We should make clear here that the only operation that works on strings is addition? I think it's kinda odd that it concats strings? Might just be me? :)

Reply:
Multiplication works as well: `2 * "foo" == "foofoo"`. I don't find it odd, but that also might just be me ;)